servlet/src/main/java/io/undertow/servlet/api/DeploymentInfo.java

```diff
@@ -96,6 +97,7 @@
     private ConfidentialPortManager confidentialPortManager;
     private boolean allowNonStandardWrappers = false;
     private int defaultSessionTimeout = 60 * 30;
+    private long defaultAsyncContextTimeout = AsyncContextImpl.DEFAULT_ASYNC_CONTEXT_TIMEOUT;
```

Comment:
Isn't `DeploymentInfo` API better not to depend on a concrete implementation like `AsyncContextImpl` as possible?
I think it's enough to have the default value of `defaultAsyncContextTimeout` (30000) in `DeploymentInfo` like `defaultSessionTimeout` does.

Reply:
if it was separate API package, Id agree. In this case Id consider this ok. For sake of transparency its good to have such values defined with jdoc, so users can look it up, rather search through source.

Reply:
@baranowb I agree with @msfm. One extra point here is keeping consistency. If we were to add final values for timeouts, we would do it in the entire code. If we already have a defaultSessionTimeout defined without a constant field in line 99 above the line you added, it makes the code more difficult to read because it fails to follow a pattern.

Reply:
I kind of dont agree, since some options follow this pattern. Just because others dont makes no sense to ignore it. Though I did update PR as requested.